data/categories.txt

```diff
@@ -5649,7 +5654,7 @@ pl:4Przedszkole|dzieci|zerówka
 pt:Jardim de infância|Jardim infantil|escola infantil|pré-escola
 pt-BR:Jardim de infância|escola infantil|pré-escola
 ro:Grădiniță
-es:Guardería
+es:Guardería|Prescolar|Jardín de infantes|Maternal
```

Comment:
Here in Mexico we also spell "prescolar" with a doubled vowel, "preescolar" (both variants are valid).

Reply:
Added.